README.md

```diff
@@ -117,13 +128,13 @@ Clone the latest version of [coins](https://github.com/KomodoPlatform/coins)
 
 Download and install the latest version of [python3](https://www.python.org/downloads/)
 
-Open the clonned repository and run the script below in the terminal in the repo folder
+Open the cloned repository and run the script below in the terminal in the repo folder.
 
 ```bash
 python3 utils/generate_app_configs.py
 ```
 
-Copy the generated `coins_config.json` file in Utils folder and paste inside assets/ folder in AtomicDEX-mobile project
+Copy the generated `coins_config.json` file from the Utils folder and paste it inside the `assets/` folder in the AtomicDEX Mobile project.
 
 ## Audio samples sources
 
```

Comment:
Please update testplan link to https://docs.google.com/spreadsheets/d/1jeIkGe2CmJ7YmuoVi6Rlc9KRr3wiBPf44Qy0Nd8qtOY/edit?usp=sharing